portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetFragment.testcase

```diff
@@ -0,0 +1,429 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property custom.properties = "feature.flag.LPS-164563=true${line.separator}feature.flag.LPS-167253=true";
+	property portal.acceptance = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Frontend Data Set";
+	property testray.main.component.name = "Frontend Data Set";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given creating a Data Set via API") {
+			DataSetAdmin.goToDataSetAdminPage();
+
+			DataSetAdmin.createDataSetEntryViaAPI(
+				dataSetName = "Data Set Test",
+				restApplication = "/data-set-manager/entries");
+
+			Refresh();
+		}
+
+		task ("When go to the View page of the data set created.") {
+			DataSetAdmin.goToViews(dataSetName = "Data Set Test");
+		}
+
+		task ("And a data set view created via api") {
+			DataSetAdmin.createDataSetView(
+				description = "Description",
+				key_name = "View Fields Test");
+		}
+
+		task ("When go to the View page of the data set created.") {
+			DataSetAdmin.goToSpecificViewPage(
+				dataSetName = "Data Set Test",
+				key_viewName = "View Fields Test");
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		DataSetAdmin.deleteAllDataSetEntries();
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-185986. Confirm that the Items Per Page values have been changed"
+	@priority = 5
+	test AssertPaginationIsChanged {
+		task ("When goes to Fields Tab.") {
+			DataSetAdmin.goToTab(tabName = "Fields");
+		}
+
+		task ("And add some fields//Go to fields tab and add") {
+			DataSetAdmin.addFields(key_fieldList = "id,status,label");
+		}
+
+		task ("When the user goes to the Pagination and assert default items") {
+			DataSetAdmin.goToTab(tabName = "Pagination");
+		}
+
+		task ("Then the default List of items per Page values are: 10, 20, 30 ,40 and default value per gage is 10") {
+			TripleClick(
```

Comment:
Why do you use a triple click method here?

Reply:
Thanks for noticing this @CarlosBrichete , I think I got confused...
but I already made the changes by removing.